src/main/kotlin/com/terning/server/kotlin/domain/user/vo/UserName.kt

```diff
@@ -22,6 +22,10 @@ class UserName(
         private const val MAX_LENGTH = 12
 
         private const val ERROR_EMPTY = "이름은 공백일 수 없습니다."
-        private val ERROR_LENGTH = "이름은 ${MIN_LENGTH}~${MAX_LENGTH}자여야 합니다."
+        private val ERROR_LENGTH = "이름은 $MIN_LENGTH~${MAX_LENGTH}자여야 합니다."
+
+        fun from(value: String): UserName {
+            return UserName(value)
+        }
```

Comment:
여기두.. 표현식이 좋을 것 같아요..!
```suggestion
        fun from(value: String): UserName = UserName(value)
```

Reply:
아이코! 이 부분은 놓치고 있었네요 😅
(아직 자바 인간인지... 표현식이 아닌 걸 봐도 어색하다는 생각이 잘 안 드네요 ㅠㅠ)
말씀 주신 대로 바로 반영하겠습니다!
꼼꼼하게 챙겨주셔서 정말 감사해요 🙏✨